lib/gpx.js

```diff
@@ -72,11 +80,28 @@ function coordPair(x) {
       ll.push(e);
     }
   }
-  return {
+  const result = {
     coordinates: ll,
     time: time ? nodeVal(time) : null,
     heartRate: heartRate ? parseFloat(nodeVal(heartRate)) : null
   };
+
+  const extensions = get1(x, "extensions");
+  if (extensions != null) {
```

Comment:
Should these be !==?

Reply:
The code is just doing a null or undefined test, it would make the test sensitive to if get1() returns undefined or null.  I'd rather not be over prescriptive in case the future behavior changes.  The old be generous handling with what you receive and specific with what you send.

Reply:
In this case, we aren't receiving the value from some external source - get1 is a private method that's part of togeojson and it explicitly always returns either an element or null.

Reply:
Should be all set with latest commits.